lib/content/audio.ex

```diff
@@ -15,10 +15,11 @@ defprotocol Content.Audio do
 
   @type language :: :english | :spanish
   @type value :: canned_message() | ad_hoc_message() | nil
+  @type tts_value :: {String.t(), [{String.t(), String.t(), integer()}] | nil}
```

Comment:
Just a clarifying question because it's relevant to my current ticket: the first String here is what gets read out and the list of pages the transcription that gets shown, which is what lets us tweak the pronunciation of the TTS from whats shown visually? Would it be helpful to add two additional types `audio_text` as `String.t()` and `visual_text` as `[{String.t(), String.t(), integer()}] | nil`  to communicate that?

Reply:
Yep, can do. Also, for your ticket, take a look at the new `paginate_text` helper if you haven't already.